src/Attribute/AuditTrailDisplayData.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Attribute;
+
+use Attribute;
+
+#[Attribute]
+class AuditTrailDisplayData
```

Comment:
Deze naam lijkt mij niet logisch?

Reply:
Dit toont de data in je AuditTrail dus lijkt me vrij duidelijk

Reply:
Dus ook veranderen naar een class attribute ipv een property attribute, want deze toont maar 1 veld

Reply:
fixed [5209f29](https://github.com/sumocoders/FrameworkCoreBundle/pull/142/commits/5209f29448655b799af03a43807a46b366b48b92)

Reply:
Hmm, nu is het dus alles of niks bijhouden van velden?

Reply:
Is indd het geval nu, anders klopt die naming niet

Reply:
Voor mij leek een logischer naam iets als `AuditTrailLoggedField` ofzo, dat moest niet voor de hele class gedaan worden

Reply:
@jonasdekeukelaere Bij deze is er nu ook een attribute om een filed te loggen 6834ea8eb953301eb7c42c8bcdfd79808800e3d0